src/components/AppNavigation/GroupNavigationItem.vue

```diff
@@ -154,6 +154,10 @@ export default {
 		emailGroup(group) {
 			const emails = []
 			group.contacts.forEach(key => {
```

Comment:
nitpick: you can add a `.filter` before `forEach` to filter out these contacts

Reply:
ok, I added a filter method